gemcalibration/include/gem/calib/Calibration.h

```diff
@@ -0,0 +1,244 @@
+          {"vfatChMin" , "VFAT Ch min"},
+          {"vfatChMax" , "VFAT Ch max"},
+          {"vt2"       , "CFG_THR_ARM_DAC"},
+          {"trigThrottle"  , "Trigger throttle (int)"},
+	  {"pulseDelay", "Pulse delay (BX)"},
+	  {"latency"   , "Latency (BX)"},
+	  {"trimValues", "Points in dac range (odd)"}, // TODO:need to be implemented properly in the back end in order to get a given number of points {-63,0,63}
+	    };
+	
+        std::map<std::string, uint32_t> m_amcOpticalLinks;
+
+	dacScanType_t m_dacScanType;
+	
+	/**
+         *  map of selectable DAC scan for the VFAT3 parameters and relative range limits 
+         */
+	
+	std::map<dacScanType_t,  std::map<std::string, uint32_t>> m_dacScanTypeParams{ 
+	  {CFG_CAL_DAC,{{"CFG_CAL_DAC_Min"  , 0},{"CFG_CAL_DAC_Max", 255},}},
+	  {CFG_BIAS_PRE_I_BIT, {{"CFG_BIAS_PRE_I_BIT_Min",0},{"CFG_BIAS_PRE_I_BIT_Max", 255},}},
+	  {CFG_BIAS_PRE_I_BLCC,{{"CFG_BIAS_PRE_I_BLCC_Min", 0},{"CFG_BIAS_PRE_I_BLCC_Max", 63},}},
+	  {CFG_BIAS_PRE_I_BSF,{{"CFG_BIAS_PRE_I_BSF_Min",0}, {"CFG_BIAS_PRE_I_BSF_Max", 63},}},
+	  {CFG_BIAS_SH_I_BFCAS,{{"CFG_BIAS_SH_I_BFCAS_Min",0},{"CFG_BIAS_SH_I_BFCAS_Max", 255},}},
+	  {CFG_BIAS_SH_I_BDIFF,{{"CFG_BIAS_SH_I_BDIFF_Min", 0},{"CFG_BIAS_SH_I_BDIFF_Max", 255},}},
+	  {CFG_BIAS_SD_I_BDIFF,{{"CFG_BIAS_SD_I_BDIFF_Min",0},{"CFG_BIAS_SD_I_BDIFF_Max",255},}},
+	  {CFG_BIAS_SD_I_BFCAS,{{"CFG_BIAS_SD_I_BFCAS_Min",0},{"CFG_BIAS_SD_I_BFCAS_Max", 255}, }},
+	  {CFG_BIAS_SD_I_BSF,{{"CFG_BIAS_SD_I_BSF_Min",0}, {"CFG_BIAS_SD_I_BSF_Max", 63},}},
+	  {CFG_BIAS_CFD_DAC_1,{{"CFG_BIAS_CFD_DAC_1_Min",0},{"CFG_BIAS_CFD_DAC_1_Max", 63},}},
+	  {CFG_BIAS_CFD_DAC_2,{{"CFG_BIAS_CFD_DAC_2_Min", 0},{"CFG_BIAS_CFD_DAC_2_Max", 63},}},
+	  {CFG_HYST,{{"CFG_HYST_Min",0},{"CFG_HYST_Max", 63},}},
+	  {CFG_THR_ARM_DAC,{{"CFG_THR_ARM_DAC_Min",0},{"CFG_THR_ARM_DAC_Max", 255},}},
+	  {CFG_THR_ZCC_DAC,{{"CFG_THR_ZCC_DAC_Min",0}, {"CFG_THR_ZCC_DAC_Max", 255},}},
+	  {CFG_BIAS_PRE_VREF,{{"CFG_BIAS_PRE_VREF_Min",0},{"CFG_BIAS_PRE_VREF_Max", 255},}},  
+	  {CFG_VREF_ADC, {{"CFG_VREF_ADC_Min",0},{"CFG_VREF_ADC_Max", 3},}}
+	    };
+
+	/**
+         *  map of selectable DAC scan for the VFAT3 parameters and relative range limits 
+         */
+	
+	std::map<dacScanType_t,  std::string > m_dacScanTypeParams_label{ 
+	  {CFG_CAL_DAC, "CFG_CAL_DAC"},
+	  {CFG_BIAS_PRE_I_BIT, "CFG_BIAS_PRE_I_BIT"},
+	  {CFG_BIAS_PRE_I_BLCC, "CFG_BIAS_PRE_I_BLCC"},
+	  {CFG_BIAS_PRE_I_BSF, "CFG_BIAS_PRE_I_BSF"},
+	  {CFG_BIAS_SH_I_BFCAS,"CFG_BIAS_SH_I_BFCAS"},
+	  {CFG_BIAS_SH_I_BDIFF, "CFG_BIAS_SH_I_BDIFF"},
+	  {CFG_BIAS_SD_I_BDIFF, "CFG_BIAS_SD_I_BDIFF"},
+	  {CFG_BIAS_SD_I_BFCAS, "CFG_BIAS_SD_I_BFCAS"},
+	  {CFG_BIAS_SD_I_BSF, "CFG_BIAS_SD_I_BSF"},
+	  {CFG_BIAS_CFD_DAC_1,"CFG_BIAS_CFD_DAC_1"},
+	  {CFG_BIAS_CFD_DAC_2,"CFG_BIAS_CFD_DAC_2"},
+	  {CFG_HYST,"CFG_HYST"},
+	  {CFG_THR_ARM_DAC,"CFG_THR_ARM_DAC"},
+	  {CFG_THR_ZCC_DAC, "CFG_THR_ZCC_DAC"},
+	  {CFG_BIAS_PRE_VREF,"CFG_BIAS_PRE_VREF"},
+	  {CFG_VREF_ADC, "CFG_VREF_ADC"},
+	};
```

Comment:
* Can all these various maps be `const`/constexpr`?
  * They seem to simply be placeholders for templates for the UI display defaults.
* Some of them seem to be duplicated unnecessarily:
  * `m_dacScanTypeParams_label` and `m_dacScanTypeParams` could be merged to give a mapping:
    * `enum` -> `{label, min, max}`